packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -446,8 +517,9 @@ function* doMarkChatAsRead(action: ReturnType<typeof markChatAsRead>) {
     const chat = yield* select((state) => getNonOptimisticChat(state, chatId))
     if (
       !chat ||
-      !chat?.last_read_at ||
-      dayjs(chat?.last_read_at).isBefore(chat?.last_message_at)
+      (!chat.is_blast &&
```

Comment:
hm this is hard to read...
but also, should it be that this entire if/else is wrapped in an outer `if !chat.is_blast`? we just don't want to do any mark as read stuff if it's a blast right?